.github/workflows/mirror.yml

```
@@ -6,6 +6,8 @@ on:
       version:
         required: true
         type: string
+    secrets:
+      - PROD_AWS_ROLE
```

actionlint (1.7.12)

[error] "secrets" section is sequence node but mapping node is expected

(syntax-check)